src/spectral.ts

```diff
@@ -1,11 +1,20 @@
 import { safeStringify } from '@stoplight/json';
 import { Resolver } from '@stoplight/json-ref-resolver';
-import { IResolveError } from '@stoplight/json-ref-resolver/types';
-import { DiagnosticSeverity, IParserResult } from '@stoplight/types';
-import { getLocationForJsonPath, parseWithPointers } from '@stoplight/yaml';
-import { merge, uniqBy } from 'lodash';
-
+import { DiagnosticSeverity, Dictionary, IParserResult } from '@stoplight/types';
+import {
+  getLocationForJsonPath,
+  getLocationForJsonPath as getLocationForJsonPathYAML,
```

Comment:
What is all this about?  :D

Reply:
Oops, oversight, no need to double the imports.